pkg/manager/index/service/indexer.go

```diff
@@ -158,12 +158,20 @@ func (idx *index) execute(ctx context.Context, enableLowIndexSkip bool) (err err
 					PoolSize: idx.creationPoolSize,
 				}, copts...)
 				if err != nil {
-					log.Debug(addr, err)
+					// TODO: handle FailedPrecondition errors using
```

Comment:
If agents return correct status code, index manager doesn't have to publish useless logs.